packages/editor/src/components/collab-sidebar/index.js

```diff
@@ -305,11 +305,16 @@ export default function CollabSidebar() {
 			updatedResult.map( ( thread ) => [ thread.id, thread ] )
 		);
 
-		const sortedComments = blockCommentIds
+		// Get comments by block order, filter out undefined threads, and exclude resolved comments.
+		const unresolvedSortedComments = blockCommentIds
 			.map( ( id ) => threadIdMap.get( id ) )
-			.filter( ( thread ) => thread !== undefined );
+			.filter( ( thread ) => thread !== undefined )
+			.filter( ( thread ) => thread.status !== 'approved' );
```

Comment:
```suggestion
			.filter( ( thread ) => thread !== undefined && thread.status !== 'approved' );
```
combine the two related conditionals to avoid iterating over the comments twice.